src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -2012,9 +2012,37 @@
       <br>
       <br>
     </li>
+    <li id="fbSessionResendLinksRequest">
+      <span class="text-bold">
+          <b>5. Requesting resending of access links</b>
+      </span>
+      <p>
+        TEAMMATES allows students to request for access links which students might accidentally deleted to be resent. Students can do so by themselves without the need to approach the instructor.
+        The student can go to the <b>Home</b> page, click <b>Request</b> tab and choose <b>Resend Feedback Links</b>, this will allow him/her to be directed to the request page. The student then keys in his/her email address and
+        all access links for the feedback sessions in which the student has participated in last six months will be sent to that email address.
+      </p>
+      <p>
+        The student can also go to the request page via url: http://teammatesv4.appspot.com/resendLinks.
```

Comment:
The URL is still being hardcoded

Reply:
Hi @whipermr5, I understand for the below one that I should not hard code. But for this information provided in the help page, isn't it need to be a real link?

Reply:
Since it's JSP, I believe it can import the value from code (i.e. `Config`)?